pkg/kubelet/dockershim/docker_service_test.go

```diff
@@ -90,3 +104,25 @@ func TestStatus(t *testing.T) {
 		runtimeapi.NetworkReady: false,
 	}, status)
 }
+
+func TestVersion(t *testing.T) {
+	ds, _, _ := newTestDockerService()
+
+	expectedVersion := &dockertypes.Version{Version: "1.11.2", APIVersion: "1.23.0"}
+	v, err := ds.getDockerVersion()
+	require.NoError(t, err)
+	assert.Equal(t, expectedVersion, v)
+
+	expectedAPIVersion := &semver.Version{Major: 1, Minor: 23, Patch: 0}
+	apiVersion, err := ds.getDockerAPIVersion()
+	require.NoError(t, err)
+	assert.Equal(t, expectedAPIVersion, apiVersion)
+}
+func TestAPIVersionWithCache(t *testing.T) {
```

Comment:
What do we want to test with this function? Should we check fake docker client to see whether `version` is actually cached?